swap/src/cli/trace.rs

```diff
@@ -9,56 +9,70 @@ use tracing_subscriber::fmt::format::{DefaultFields, Format};
 use tracing_subscriber::fmt::time::ChronoLocal;
 use tracing_subscriber::layer::{Context, SubscriberExt};
 use tracing_subscriber::registry::LookupSpan;
-use tracing_subscriber::{fmt, Layer, Registry};
+use tracing_subscriber::{fmt, EnvFilter, Layer, Registry};
 use uuid::Uuid;
 
 pub fn init_tracing(
     debug: bool,
     dir: impl AsRef<Path>,
     command: &Command,
+    new_swap_id: &Uuid,
```

Comment:
This is kind of confusing IMO because the `new_swap_id` is only used if the command is `XmrBtc`. Can we clean this up so that we are not passing redundant arguments?

Reply:
Hmmm I don't like this either :( but I think it is bit more involved than a clean up.
We could add `swap_id` to `Command::BuyXmr`. Need to figure out how to get structopt to ignore it when parsing args